src/components/NavBar.jsx

```diff
@@ -1,27 +1,45 @@
 import { NavLink, Link } from 'react-router-dom';
+import { useState } from 'react';
 import logo from '../images/planet_1.png';
 
-const NavBar = () => (
-  <header>
-    <nav>
-      <Link to="/">
-        <img className="logo" src={logo} alt="" />
-        <h1>Space traveler&apos; Hub</h1>
-      </Link>
-      <ul>
-        <NavLink className="nav-items" to="/">
-          Rockets
-        </NavLink>
-        <NavLink className="nav-items" to="/missions">
-          Missions
-        </NavLink>
-        <span>|</span>
-        <NavLink className="nav-items" to="/profile">
-          My Profile
-        </NavLink>
-      </ul>
-    </nav>
-  </header>
-);
+const NavBar = () => {
+  const [active, setActive] = useState('Rockets');
+
+  return (
+    <header>
+      <nav>
+        <Link to="/">
+          <img className="logo" src={logo} alt="" />
+          <h1>Space traveler&apos; Hub</h1>
+        </Link>
+        <ul>
+          <NavLink
+            className={active === 'Rockets' ? 'active' : ''}
```

Comment:
[Optional] Greate job to add style active class to indicate which section/page the user is currently on. But better if you only manage it with CSS style in order to avoid event firing every time the link is clicked.

`const navLinkStyle = ({ isActive }) => {
    const activeLinkStyle = {
      fontWeight: isActive ? 'bold' : 'normal',
      textDecoration: isActive ? 'underline' : 'none',
    };
    return activeLinkStyle;
  };

NavLink to="/" className="navLink" style={navLinkStyle}>
    Rockets
</NavLink>`

Reply:
@cherelemma  nice catch.
Fixed; have a look at it [here](https://github.com/ZewdieMc/space-traveler/pull/53/commits/ee7f4dda260f16c825b83fa36d1cfc79c9efa7fc).